aspnetcore/includes/vs-mac-eol.md

```diff
@@ -0,0 +1,14 @@
+---
+author: tdykstra
+ms.author: tdykstra
+ms.date: 09/15/2023
+ms.topic: include
```

Comment:
Putting include files in a `/include` folder is a good convention, but I'm wondering with `ms.topic: include` if that's not a requirement.